docs/dg/dev/supported-versions-of-php.md

```diff
@@ -11,25 +11,25 @@ redirect_from:
 
 ## PHP 8
 
-Since December 2023, Spryker has been compatible with PHP 8.2. Support for PHP 8.3 was added in September 2024.
+Since August 2025, Spryker has been compatible with PHP 8.4. Support for PHP 8.3 was added in September 2024.
 
-Currently, we *recommend* using *PHP 8.3*. If you are using version 8.2, we encourage you to migrate at the end of 2024 to ensure a smooth upgrade path.
-More and more libraries deliver releases, including bug fixes and security updates, only for 8.3+, so this becomes an increasing risk factor for any business.
+Currently, we *recommend* using *PHP 8.4*. If you are using version 8.3, we encourage you to migrate at the end of 2025 to ensure a smooth upgrade path.
```

Comment:
Well, it seems that we communicated to migrate to 8.3 till the end of year and not in the middle of it...

In any case, I would make it 2026 in this case.